HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py

```diff
@@ -476,6 +477,7 @@
         'keep *_hltDeepCombinedSecondaryVertexBJetTagsInfos_*_*',
         'keep *_hltDeepCombinedSecondaryVertexBJetTagsPF_*_*',
         'keep *_hltDeepSecondaryVertexTagInfosPF_*_*',
+        'keep *_hltParticleNet*_*_*',
```

Comment:
This file is automatically generated, so these changes will be lost. Please remove them and instead file a CMSHLT JIRA request and explanation why this addition is needed, I note the wildcard in the keep adding all module instances starting with ```hltParticleNet*```